doc/release-notes.rst

```diff
@@ -2,6 +2,245 @@
  Release Notes
 ===============
 
+v12.2.3 Luminous
+================
+
+This is the second bugfix release of Luminous v12.2.x long term stable release
+series. It contains a range of bug fixes and a few features across Bluestore,
+CephFS, RBD & RGW. We recommend all the users of 12.2.x series update.
+
+For more detailed information, see :download:`the complete changelog <changelog/v12.2.2.txt>`.
+
+
+Notable Changes
+---------------
+
+- *CephFS*:
+
+  * The CephFS client now catches failures to clear dentries during startup
+    and refuses to start as consistency and untrimmable cache issues may
```

Comment:
The CephFS client now checks for older kernels' inability to reliably clear dentries from the kernel dentry cache